docs/references/api.md

```diff
@@ -801,3 +824,104 @@ pro api u.security.package_manifest.v1
     "package_manifest":"package1\t1.0\npackage2\t2.3\n"
 }
 ```
+
+## u.unattended_upgrades.status.v1
+
+Introduced in Ubuntu Pro Client Version: `27.14~`
+
+Returns the status around unattended-upgrades. The focus of the endpoint
+is to verify if the application is running and how it is configured on
+the machine
+
+```{important}
+For this endpoint, we deliver an unique key under `meta` called `raw_config`. This field contains
+all related unattended-upgrades configurations unparsed. This means that this field with maintain
```

Comment:
```suggestion
all related unattended-upgrades configurations unparsed. This means that this field will maintain
```

Reply:
Not sure if this is the correct suggestion here, but I think "with" might be a typo

Reply:
That's a typo, thanks for spotting that